pkgs/development/python-modules/gruut-ipa/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pkgs
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "gruut-ipa";
+  version = "0.9.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "rhasspy";
+    repo = pname;
+    rev = "df74f7dff562d868ad8088b7251f9e7206271d37";
```

Comment:
You should file an issue about missing tags.

Reply:
As soon as I get `python-crfsuite` running. If I don't then :man_shrugging:.

Reply:
https://github.com/rhasspy/gruut-ipa/issues/3